Resources/Prototypes/_LostParadise/Entities/Structures/Wallmounts/posters.yml

```
@@ -45,4 +45,74 @@
+  description: "You're on the wrong side of Heaven my friend"
+  components:
+  - type: Sprite
+    state: lustfulvulp-press
+
+- type: entity
+  parent: LPPPosterBase
+  id: LPPPosterLustfulVulp1
+  name: "Horny vulp"
+  description: "Run for your life!"
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)